src/main/scala/io/druid/indexer/spark/SparkBatchIndexTask.scala

```diff
@@ -259,6 +261,8 @@ object SparkBatchIndexTask
   private val DEFAULT_TARGET_PARTITION_SIZE: Long   = 5000000L
   private val CHILD_PROPERTY_PREFIX        : String = "druid.indexer.fork.property."
   val log            = new Logger(SparkBatchIndexTask.getClass)
+  val lifecycle      = SerializedJsonStatic.lifecycle
```

Comment:
and I'm not sure if the lifecycle should be started here or if it is expected to be handled elsewhere.